extensions/amp-story/1.0/page-advancement.js

```diff
@@ -462,6 +462,8 @@ export class ManualAdvancement extends AdvancementConfig {
       (el) => {
         tagName = el.tagName.toLowerCase();
 
+        // Prevents navigatin when clicking inside draggable drawers including:
+        // amp-story-page-attachment, amp-story-page-outlink and amp-story-shopping-attachment.
```

Comment:
Maybe tweaking the comment to something like "Prevents navigation when clicking inside of draggable drawer elements, such as `<amp-story-page-attachment>` and `<amp-story-page-outlink>`." would prevent us from needing to update this list of draggable drawer types when we introduce another type in the future? Up to you!

Reply:
Sounds good to make it more general :) Updated.